src/modules/selection/RectangularSelection.jsx

```
@@ -91,8 +96,10 @@ const RectangularSelection = ({
   const [isContainerReady, setIsContainerReady] = useState(false)
   const { setSelectedItems, selectedItems, setIsSelectAll } =
     useSelectionContext()
+  const scrollableContainer = scrollElement || scrollContainerRef?.current
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

   97 |   const { setSelectedItems, selectedItems, setIsSelectAll } =
   98 |     useSelectionContext()
>  99 |   const scrollableContainer = scrollElement || scrollContainerRef?.current
      |                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  100 |   const isDraggingRef = useRef(false)
  101 |   const dragStartPosRef = useRef(null)
  102 |   const wheelScrolledDuringDragRef = useRef(false)

(react-hooks/refs)